receiver/awsxrayreceiver/README.md

```diff
@@ -0,0 +1,73 @@
+# AWS X-Ray Receiver
+
+**Status: alpha**
+
+## Overview
+The AWS X-Ray receiver accepts segments (i.e. spans) in the [X-Ray Segment format](https://docs.aws.amazon.com/xray/latest/devguide/xray-api-segmentdocuments.html).
+This enables the collector to receive spans emitted by the existing X-Ray SDK. [Centralized sampling](https://github.com/aws/aws-xray-daemon/blob/master/CHANGELOG.md#300-2018-08-28) is also supported via a local TCP port.
+
+## Configuration
+
+Example:
+
+```yaml
+receivers:
+  aws_xray:
+    version: 1.0.0
+    endpoint: localhost:2000
+    proxy_server:
```

Comment:
`proxy_server` doesn't seem to be documented, I assumed all the fields were top-level

Reply:
And actually, shouldn't they all be top-level? We use the X-Ray API for both segments (from endpoint) and sampling (from tcp_endpoint), all the other fields apply to using the X-Ray API, not just the proxy I believe.

Reply:
Ah that's right got confused by thinking about the daemon's workings.